app/server/src/interfaces/roles.interface.ts

```diff
@@ -15,11 +15,11 @@
  */
 
 export enum Role {
-  Admin = 'admin',
-  Submitter = 'submitter',
-  FA = 'fa', // financial analyst
-  Executive = 'executive',
-  User = 'user',
+  Admin = 'Admin',
+  Submitter = 'Submitter',
+  FA = 'FA', // financial analyst
```

Comment:
Is there a reason we just use FA? I think Finance Analyst would read more clearly to the end users.